app/Resources/views/base.mobile.html.twig

```diff
@@ -0,0 +1,109 @@
+                                </div>
+                            {% endif %}
+                        </span>
+                </div>
+                        
+                {% if not is_granted('ROLE_USER') %}
+                 <div class="menulist">                     
+                       <div class="menuelement" onclick='move("{{ path('fos_user_registration_register') }}");'>
+                         <img src="{{ asset('images/icons/registration.png') }}"/>
+                         <span> {{ 'registration' | trans }}</span>
+                     </div>
+                 </div>
+                 {% endif %}
+                 
+                <div class="menulist">
+                     <div class="menuelement" onclick='move("{{ path('homepage') }}");'>
+                         <img src="{{ asset('images/icons/home.png') }}"/>
+                         <span> {{ 'homepage' | trans }}</span>
+                     </div>
+                    {% if is_granted('ROLE_ADMIN') %}
+                    <div class="menuelement" onclick='move("{{ path('admin_dashboard') }}");'>
+                         <img src="{{ asset('images/icons/adminspace.png') }}"/>
+                         <span> {{ 'admin.dashboard' | trans }}</span> 
+                    </div>
+                    {% endif %} 
+                    {% if is_granted('ROLE_USER') %} 
+                         <div class="menuelement" onclick='move("{{ path('dashboard') }}");'>
+                             <img src="{{ asset('images/icons/myspace.png') }}"/>
+                             <span> {{ 'user.dashboard' | trans }}</span>
+                         </div>
+                    {% endif %}
+                    {% if is_granted('ROLE_USER') %}
+                          <div class="menuelement" onclick='move("{{ path('homepage') }}");'>
+                              <img src="{{ asset('images/icons/notification.png') }}"/>
+                              <span> {{ 'notifcation.all' | trans }}</span> 
+                          </div>
+                    {% endif %}
+                    <div class="menuelement" onclick='move("{{ path('project_board') }}");'>
+                        <img src="{{ asset('images/icons/project.png') }}" />
+                        <span>{{ 'project.board' | trans }}</span>
+                    </div>
+                </div>
+                 <div class="menulist">
+                      {% if is_granted('ROLE_USER') %}
+                        <div class="menuelement" onclick='move("{{ path('homepage') }}");'>
+                            <img src="{{ asset('images/icons/myaccount.png') }}" />
+                            <span>{{ 'user.account' | trans }}</span>
+                        </div>
+                    {% endif %}
+                         
+                 </div>
+                
+            </div>  
+
+            <div id="page-title"  >
+                   <button id="menu_button" onclick="showMenu()">Menu</button>
```

Comment:
I think we can keep the desktop three bars button for menu toggling, mobile users are familiar with it